doc/source/serve/autoscaling-guide.md

```diff
@@ -0,0 +1,369 @@
+| QPS | <img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_rps.svg" alt="qps" width="450"/> |
+| P50 Latency | <img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_latency.svg" alt="latency" width="450"/> |
+
+
+<!-- | ![replicas](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_replicas.png){width=200px} | ![qps](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_rps.png){width=200px} | ![latency](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_latency.svg){width=200px} | -->
+
+Notice that the number of autoscaled replicas closely matches the number of Locust users over time. This makes sense because you can simplify it to one user is mapped to one deployment replica, and the user will continuously send the right amount of traffic to keep one replica busy. As expected, the throughput of the system also increases with the number of users and replicas. Lastly, the latency sees some spikes when the traffic increases, but otherwise stays relatively steady. Notice that the biggest latency spike happens at the beginning of the test, because the deployment starts with 0 replicas.
+
+
+## Model Composition Example
+
+It is less straightforward to determine an autoscaling configuration for a multi-model application because different deployments will autoscale differently. Every workload will have a different latency and differing levels of concurrency. As a result, it takes experimentation to find the right autoscaling config for a model-composition application.
+
+We will go through an example of a simple application with three deployments composed together to build some intuition about multi-model autoscaling. Suppose we have deployments:
+* `HeavyLoad`: A mock 200ms workload with high CPU usage.
+* `LightLoad`: A mock 100ms workload with high CPU usage.
+* `Driver`: A driver deployment that fans out to the `HeavyLoad` and `LightLoad` deployments and aggregates the two outputs.
+
+
+### Attempt 1: One `Driver` replica
+
+Let's start off with the following deployment configurations. Since the driver deployment has low CPU usage and is only asynchronously making calls to the downstream deployments, it's reasonable to allocate one fixed `Driver` replica.
+
+::::{tab-set}
+
+:::{tab-item} Driver
+
+```yaml
+- name: Driver
+  num_replicas: 1
+  max_concurrent_queries: 200
+```
+
+:::
+
+:::{tab-item} HeavyLoad
+
+```yaml
+- name: HeavyLoad
+  max_concurrent_queries: 3
+  autoscaling_config:
+    target_num_ongoing_requests_per_replica: 1
+    min_replicas: 0
+    initial_replicas: 0
+    max_replicas: 200
+    upscale_delay_s: 3
+    downscale_delay_s: 60
+    upscale_smoothing_factor: 0.3
+    downscale_smoothing_factor: 0.3
+    metrics_interval_s: 2
+    look_pack_period_s: 10
```

Comment:
We never mention these settings until now. Can we either define them or add a note with a link to where their definitions are?

Reply:
I added a reference to the section below!